internal/pkg/flink/internal/controller/input_controller_test.go

```diff
@@ -1,35 +1,80 @@
 package controller
 
 import (
+	"errors"
+	"fmt"
 	"net/http"
+	"os"
 	"testing"
 
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/confluentinc/cli/internal/pkg/flink/test/mock"
 	"github.com/confluentinc/cli/internal/pkg/flink/types"
 )
 
-func TestRenderError(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	mockAppController := mock.NewMockApplicationControllerInterface(ctrl)
+type InputControllerTestSuite struct {
+	suite.Suite
+	mockAppController   *mock.MockApplicationControllerInterface
+	mockTableController *mock.MockTableControllerInterface
+	mockPrompt          *mock.MockIPrompt
+	mockStore           *mock.MockStoreInterface
+}
+
+func TestInputControllerTestSuite(t *testing.T) {
+	suite.Run(t, new(InputControllerTestSuite))
+}
+
+func (s *InputControllerTestSuite) runAndCaptureSTDOUT(test func()) string {
+	// Redirect STDOUT to a buffer
+	old := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
```

Comment:
nit for later: does go prompt allow you to inject stdout in any way?

Reply:
Not easily at least, I think you would have to implement a custom version of the `ConsoleWriter` interface and then pass that when constructing your Prompt struct: https://github.com/confluentinc/go-prompt/blob/master/option.go#L295